### PR TITLE
Quality assurance testPlan

### PR DESCRIPTION
**Pull request by Furimizu**
Test plan in main

### CODE REVIEW

File: Documents/QualityAssurance/testPlan.md [hunk 1]
```diff
@@ -1 +1,214 @@
...
+- [I . Introduction](#i-introduction)
+    - [1 . Purpose of the Test Plan](#1-purpose-of-the-test-plan)
+    - [2 . Product Overview](#2-product-overview)
+    - [3 . Targeted Audience](#3-targeted-audience)
+        - [a . Primary Audience](#a-primary-audience)
+        - [b . Secondary Audience](#b-secondary-audience)
+- [II . Testing Plan](#ii-testing-plan)
+    - [1 . Types of Testing](#1-types-of-testing)
+        - [a . Playtesting](#a-playtesting)
+        - [b . Functional Testing](#b-functional-testing)
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:44]: dead link

File: Documents/QualityAssurance/testPlan.md [hunk 2]
```diff
@@ -1 +1,214 @@
...
+        - [c . Unit Testing](#c-unit-testing)
+    - [2 . Scope of Testing](#2-scope-of-testing)
+        - [a . In Scope](#a-in-scope)
+        - [b . Out of Scope](#b-out-of-scope)
+        - [c . Goal of Testing](#c-goal-of-testing)
+    - [3 . Test Environment](#3-test-environment)
+        - [a . Hardware](#a-hardware)
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:51]: dead link

File: Documents/QualityAssurance/testPlan.md [hunk 3]
```diff
@@ -1 +1,214 @@
...
+        - [b . Software](#b-software)
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:52]: dead link

File: Documents/QualityAssurance/testPlan.md [hunk 4]
```diff
@@ -1 +1,214 @@
...
+        - [c . Testing Tools](#c-testing-tools)
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:53]: dead link

File: Documents/QualityAssurance/testPlan.md [hunk 5]
```diff
@@ -1 +1,211 @@
...
+### 3-Test-Environment
+
+#### a . Hardware:
+
+The following is a comprehensive list of the various types of hardware that will be utilized during the testing process.
+Each hardware plays a crucial role in ensuring the accuracy, reliability, and overall performance of the system under test.
+
+| Computer | OS |
+| -- | -- |
+| [MacBook Air M3](https://www.apple.com/fr/shop/buy-mac/macbook-air/13-pouces-minuit-puce-apple-m3-avec-cpu-8-c%C5%93urs-et-gpu-10-c%C5%93urs-24-go-de-m%C3%A9moire-512go?cid=aos-fr-seo-pla-mac-mac) | MacOS |
```
Comment (EnzoGuillouche) [inline Documents/QualityAssurance/testPlan.md:141]: specify the macOS version as well (e.g. Sequoia)

File: Documents/QualityAssurance/testPlan.md [hunk 6]
```diff
@@ -1 +1,211 @@
...
+| [ThinkBook 14 G4](https://www.lenovo.com/fr/fr/p/laptops/thinkbook/thinkbook-series/thinkbook-14-gen-4-14-inch-intel/len101b0012?orgRef=https%253A%252F%252Fwww.google.com%252F&srsltid=AfmBOopBOFr6V3tso5XgeaO93pHfZ6EcHRxYFLVsx8v8W9MFWoyUkbq0) | Windows 11 |
+| IdeaPad slim 1-14AST-05 | Linux Ubuntu |
+
+#### b . Software:
+
+The software utilized for this project is the [Godot](https://godotengine.org/) Engine, a powerful and versatile open-source game development platform.
```
Comment (EnzoGuillouche) [inline Documents/QualityAssurance/testPlan.md:147]: precise the Godot version (4.3)

File: Documents/QualityAssurance/testPlan.md [hunk 7]
```diff
@@ -1 +1,214 @@
...
+**thoroughly verified** against the specified requirements. This document plays a key role in identifying **potential risks**, 
+resolving **defects**, and validating the software’s **readiness for deployment**.
+
+Additionally, the test plan supports the broader project goal of delivering a **high-quality**, **secure**, and **user-friendly product**. 
+By serving as a **comprehensive guide**, it provides **clear instructions** for the QA team and acts as a **reference** for project managers 
+and stakeholders involved in the testing process.
+
+### 2-Product-Overview
+
+The project is a **serious game about ecology** developed using the **Godot Engine**, designed to educate and engage users on critical environmental topics.
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:83]: it's a serious game about climat changes, not ecology

File: Documents/QualityAssurance/testPlan.md [hunk 8]
```diff
@@ -1 +1,214 @@
...
+
+#### c-Goal-of-Testing:
+The primary goal of testing our serious game is to ensure that the game effectively **educates players about ecology** while providing an **engaging and enjoyable experience**. This involves **verifying the educational validity and clarity of ecological concepts**, assessing the overall **user experience for intuitiveness and enjoyment**, evaluating the game's ability to **engage and motivate players**, testing for **technical issues such as bugs and performance lags**, ensuring **accessibility for all players**, measuring the **effectiveness of learning outcomes**, and collecting **feedback to identify areas for improvement**. By achieving these objectives, the testing process will confirm that **"Forsaken Planet"** is both a fun, engaging game and an effective educational tool for beginners interested in ecology.
```
Comment (MistzSoftware) [inline Documents/QualityAssurance/testPlan.md:130]: it still not ecology but climat changes

File: Documents/QualityAssurance/testPlan.md [hunk 9]
```diff
@@ -1 +1,211 @@
...
+
+### 2-Test-Cases-and-Test-Reports:
+**Test cases** are detailed, step-by-step instructions for executing specific tests to verify the functionality and performance of the game. They include preconditions, test steps, expected results, and any necessary data. **Test reports** document the outcomes of these test cases, providing a comprehensive overview of the testing process. These reports include information on the tests executed, the results obtained, any defects identified, and the overall status of the testing effort. The test cases and test reports are crucial for ensuring that the game meets all quality and functionality benchmarks. The test cases can be accessed [here](https://docs.google.com/spreadsheets/d/1QlrnAxoVu1ugK-oe6ddaVtVt9Ms-K2OidJbMQZVLbaQ/edit?gid=0#gid=0).
+
+## VI-Glossary-of-Terms
+
+| Term | Definition | More Information |
+|------|------------|------------------|
+| **Godot** | An open-source game engine used for developing 2D and 3D games. | [Godot Engine](https://godotengine.org/) |
```
Comment (EnzoGuillouche) [inline Documents/QualityAssurance/testPlan.md:208]: same with the Godot version

File: Documents/QualityAssurance/testPlan.md [hunk 10]
```diff
@@ -1 +1,211 @@
...
+| **Software** | The programs and other operating information used by a computer. | [Tech Target](https://www.techtarget.com/searchapparchitecture/definition/software#:~:text=Software%20is%20a%20set%20of,that%20run%20on%20a%20device.)|
+| **Hardware** | The physical and electronic parts of a computer, rather than the instructions it follows | [Cambridge Dictionary - Hardware](https://dictionary.cambridge.org/dictionary/english/hardware) |
+| **Quality Assurance** | The maintenance of a desired level of quality in a service or product, especially by means of attention to every stage of the process of delivery or production. | [Cambridge Dictionary - Quality Assurance](https://dictionary.cambridge.org/dictionary/english/quality-assurance) |
```
Comment (EnzoGuillouche) [inline Documents/QualityAssurance/testPlan.md:211]: useless to put this

**Review by EnzoGuillouche — CHANGES_REQUESTED** [review]
Some changes need to be done.

**Review by MistzSoftware — CHANGES_REQUESTED** [review]
few things to modify

File: Documents/QualityAssurance/testPlan.md (hunk 2 above)
Comment (MistzSoftware) [reply Documents/QualityAssurance/testPlan.md:51]: still broken

File: Documents/QualityAssurance/testPlan.md (hunk 3 above)
Comment (MistzSoftware) [reply Documents/QualityAssurance/testPlan.md:52]: still broken

File: Documents/QualityAssurance/testPlan.md (hunk 4 above)
Comment (MistzSoftware) [reply Documents/QualityAssurance/testPlan.md:53]: still broken

**Review by MistzSoftware — CHANGES_REQUESTED** [review]

**Review by EnzoGuillouche — APPROVED** [review]
Issues resolved. Merging approved.

**Review by MistzSoftware — APPROVED** [review]
Everything seems good

**Pull request merged**